nest/src/RoycoNestMarketHelper.sol

```diff
@@ -0,0 +1,482 @@
+        }
+
+        emit VaultAdded(identifier);
+    }
+
+    /**
+     * @dev Remove a vault configuration (deactivate it)
+     */
+    function removeVault(
+        string memory identifier
+    ) external onlyRole(ADMIN_ROLE) {
+        RoycoMarketHelperStorage storage $ = _getRoycoMarketHelperStorage();
+
+        if (!$.vaults[identifier].active) {
+            revert VaultNotActive(identifier);
+        }
+
+        $.vaults[identifier].active = false;
+        emit VaultRemoved(identifier);
+    }
+
+    /**
+     * @dev Update an existing vault configuration
+     */
+    function updateVault(
+        string memory identifier,
+        address teller,
+        address vault,
+        address accountant,
+        uint256 slippageBps,
+        uint256 performanceBps
+    ) external onlyRole(ADMIN_ROLE) {
+        RoycoMarketHelperStorage storage $ = _getRoycoMarketHelperStorage();
+
+        // If the vault doesn't exist in our tracking, add it
+        if (!$.vaultExists[identifier]) {
+            $.vaultIdentifiers.push(identifier);
+            $.vaultExists[identifier] = true;
+        }
+
+        // Update the vault
+        $.vaults[identifier] = BoringVault({
+            teller: ITeller(teller),
+            vault: IBoringVault(vault),
+            accountant: IAccountantWithRateProviders(accountant),
+            slippageBps: slippageBps,
+            performanceBps: performanceBps,
+            active: true
+        });
+
+        emit VaultUpdated(identifier);
+    }
+
+    /**
+     * @dev Update fees for a vault
+     */
+    function updateVaultFees(
+        string memory identifier,
+        uint256 slippageBps,
+        uint256 performanceBps
+    ) external onlyRole(ADMIN_ROLE) {
+        RoycoMarketHelperStorage storage $ = _getRoycoMarketHelperStorage();
+
+        if (!$.vaults[identifier].active) {
+            revert VaultNotActive(identifier);
+        }
+
+        $.vaults[identifier].slippageBps = slippageBps;
+        $.vaults[identifier].performanceBps = performanceBps;
+
+        emit VaultUpdated(identifier);
+    }
+
+    /**
+     * @dev Get vault details
+     */
+    function vaults(
+        string memory identifier
+    )
+        external
+        view
+        returns (
+            address teller,
+            address vault,
+            address accountant,
+            uint256 slippageBps,
+            uint256 performanceBps,
+            bool active
+        )
+    {
+        RoycoMarketHelperStorage storage $ = _getRoycoMarketHelperStorage();
+        BoringVault storage boringVault = $.vaults[identifier];
+
+        return (
+            address(boringVault.teller),
+            address(boringVault.vault),
+            address(boringVault.accountant),
+            boringVault.slippageBps,
+            boringVault.performanceBps,
+            boringVault.active
+        );
+    }
+
+    /**
+     * @dev Calculate minimum mint amount based on vault parameters
```

Comment:
Specify that `depositAmount` is in tokens and not in wei, could you double check with Royco that's true?